source/html/renderer/src/card-elements.ts

```diff
@@ -3405,7 +3412,7 @@ class InlineAdaptiveCard extends AdaptiveCard {
     }
 }
 
-var defaultHostConfig: HostConfig.IHostConfig = {
+var defaultHostConfig: HostConfig.HostConfig = {
```

Comment:
Now that HostConfig is a class, this defaultHostConfig variable needs to go. The "default" host config should be whatever the default values of a new instance of HostConfig are. In other words, you need to initialize all fields of the HostConfig class (either as they are declared or in its constructor.) As a developer, I get a default host config by writing

```
var defaultHostConfig = new HostConfig();
```